paprika/align.py

```diff
@@ -192,3 +192,158 @@ def offset_structure(structure, offset):
     structure.coordinates = offset_coords
     logger.info("Added offset of {} to atomic coordinates...".format(offset))
     return structure
+
+
+def translate_to_origin(structure, weight="mass", atom_mask=None, dim_mask=None):
+    """Translate a structure to the origin based on the center of 
+       mass of the whole system or of a particular choice of atom(s).
+
+    Parameters
+    ----------
+    structure : str or parmed.Structure
+        Molecular structure containing coordinates.
+    weight : str
+        Calculate the center based on atomic masses ('mass' default) or geometric center ('geo')?
+    atom_mask : str
+        Selection of atom(s) if a particular subset is preferred to estimate the center.
+    dim_mask : list
+        A mask that will filter the dimensions to which the translation will be applied (default is [1,1,1]).
+
+    Returns
+    -------
+    structure : parmed.Structure
+        A molecular structure with the coordinates translated to the origin.
+
+    """
+    # Check if weight variable is properly chosen
+    if weight not in ["mass", "geo"]:
+        raise SystemExit('Error: "weight" must either be "mass" or "geo"')
```

Comment:
I don't think we want `SystemExit` here. I'm not sure what this would do in a `jupyter notebook`. It might kill the whole kernel. Maybe `KeyError` or something like that?  Or `NameError`.

Reply:
Changed to Exception.